### PR TITLE
Fixes #28 and probably #29 as well

### LINKED ISSUE #28: Switch from xml2json to xml2js
[xml2js](https://www.npmjs.com/package/xml2js) is a parser written in pure JavaScript so the node-gyp buildstep we get by using xml2json will no longer be necessary.

Haven't looked into if we can just switch or if it requires a lot of extra work.

### PR DESCRIPTION
**Pull request by zrrrzzt**
Removes xml2json and implements xml2js

### CODE REVIEW

**Comment by zrrrzzt** [comment]
This passes all test locally. It looks like the reason it fails on Travis is line 5 in .travis.yml.
It is a unnecessary step given eslint and nodeunit are in the dev dependencies.

File: package.json [hunk 1]
```diff
@@ -64,4 +64,4 @@
     "nodeunit": "^0.9.1",
     "verb": "~0.2.0"
   }
-}
+}
```
Comment (anselmbradford) [inline package.json:67]: Would it make sense to keep the trailing newline?

File: index.js [hunk 2]
```diff
@@ -34,39 +35,45 @@ function generateReport(xml) {
...
-        solution: result.repair
-      });
-    }
-    if (result.resultType === 'Potential Problem') {
-      warnings.push({
-        line: result.lineNum,
-        column: result.columnNum,
-        message: getErrorMsg(result.errorMsg),
-        source: result.errorSourceCode
+      results = report.results.result instanceof Array
```
Comment (anselmbradford) [inline index.js:49]: What ES environment are we in here. Is [`isArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) available?

File: package.json (hunk 1 above)
Comment (zrrrzzt) [inline package.json:67]: Not for a .json-file

File: index.js (hunk 2 above)
Comment (contolini) [inline index.js:49]: As of 0.10.x, node.js is fully es5-compliant.

Comment (zrrrzzt) [inline index.js:49]: The engine is set for >=0.8 so that would be 5 and isArray is available.

File: package.json (hunk 1 above)
Comment (anselmbradford) [inline package.json:67]: Yeah this has never been [clear to me](https://github.com/lytics/ember-cli-release/issues/21#issuecomment-133497274). Minor and fine either way.

**Comment by contolini** [comment]
Looks good to me, @zrrrzzt. I'll open a PR to remove the unnecessary .travis.yml global installs.

**Comment by contolini** [comment]
Thanks for helping review, @anselmbradford

**Comment by anselmbradford** [comment]
Added https://github.com/cfpb/node-wcag/releases/tag/v0.2.0

**Pull request merged**